build_tools/benchmarks/suites/cmake_rule_generator.py

```diff
@@ -0,0 +1,204 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Generator that generates CMake rules from python defined benchmarks.
```

Comment:
(I might have missed this in an earlier review / design doc)
Are the generated CMake rules going to be checked in, or just materialized when building benchmarks?

Reply:
I haven't fully decided yet. But I believe it's possible to generate and include a CMakefile during the configuration? I'll prefer not to check in the generated CMakefile.

Reply:
Hmm.. seems tricky but possible: https://stackoverflow.com/questions/52556785/code-generator-generating-its-own-cmake-files-and-targets

Reply:
Hmm that seems a bit scary into deep CMake land, which is not a nice land